tests/nonbonded/test_nonbonded_mol_energy.py

```diff
@@ -245,7 +245,7 @@ def test_nonbonded_mol_energy_matches_exchange_mover_batch_U_in_complex(precisio
         bound_impls,
         movers=[baro_impl],
     )
-    ctxt.multiple_steps(1000)
+    ctxt.multiple_steps(5000)
```

Comment:
(This test was failing - after some offline discussion with @badisa  we're going to temporarily hack around this to unblock the PR and do a more systematic review of our energy tests later on).